### ISSUE TITLE
AudioStreamOGGVorbis prints an error message when created in the editor

### ISSUE THREAD

**Issue by Teashrock** [godotengine/godot#34734]
**Godot version:**
85fa17d3c

**OS/device including version:**
Windows 10 x64

**Issue description:**
When a "pure" AudioStreamOGGVorbis is created from menu, it spawns an error of this kind:
```
modules\stb_vorbis\audio_stream_ogg_vorbis.cpp:132 - Condition ' data == 0 ' is true. returned: ovs

```

**Steps to reproduce:**
1. Create and select any kind of AudioStreamPlayer.
2. Click the "Stream" segment of the correspondent menu.
3. Try to select "New AudioStreamOGGVorbis".
4. Get an error and no created stream at all.

**Minimal reproduction project:**
http://www.mediafire.com/file/3mys7gk2w9pntvm/AudioStreamOGGVorbis_Bug_Test.zip/file

**Comment by bruce965** [comment]
Related https://github.com/godotengine/godot/issues/16219

> Basically, you should have used `Load`, not `New` _[…]_ You can theoretically create any kind of resource from the editor, but not all of them make sense to be assigned using `New`.

**Comment by KoBeWi** [comment]
Still valid in 3.2.4 beta4
I think the error message should just mention that you should not create the resource manually. Or maybe it should not be possible.

**Comment by Teashrock** [comment]
@KoBeWi 
Or maybe just mark the new and pure resource as "to be filled with data". It should not be an error. Why not allow user to assign data at any time they want?

**Comment by KoBeWi** [comment]
Resolved by #44986
